activesupport/lib/active_support/deprecation/reporting.rb

```diff
@@ -36,6 +37,16 @@ def silence
         @silenced = old_silenced
       end
 
+
+      # Outputs a deprecation warning with 'will be removed' message to
+      # the output configured by <tt>ActiveSupport::Deprecation.behavior</tt>.
+      # By default +caller_backtrace+ is equal to caller(2).
+      #
+      #   ActiveSupport::Deprecation.deprecation_warning('something broke!')
```

Comment:
> What do you think if i use 'deprecated method name' instead 'something broke!'?

No, that's still not an *example*. An example would look like an actual method name. (The same applies to 'your message', really.)

> For me output value is more important

I agree that if we're going to document something, it should be the log output -- as far as I can see, this method has no documented return value.

What I asked, however, is whether there are other places we've used `=>`, which is normally reserved to mean "here is the return value", to instead mean "and here's some other thing". If I were reading this documentation, what indication does it give me that it's *not* the return value?